sources/libc/ansi/errno.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+/* --- 7.5 Errors <errno.h> ----------------------------------------------- */
+
+int EDOM;
+
+int EILSEQ;
+
+int ERANGE;
+
+int errno;
```

Comment:
you should set errno to zero on entry function

Reply:
I made some changes it is okay now?